src/frontend/src/routes/+page.svelte

```diff
@@ -16,7 +16,7 @@
     (is_paid_story && !(credentials.username && credentials.password));
 
   $: url =
-    `/download/${story_id}?om=1` +
+    `/download/${story_id.replaceAll("/","")}?om=1` +
```

Comment:
Actually, let's change _this_ (param on the frontend) to Story URL, and do all the extraction through javascript.

This way, the API stays unchanged.